### ISSUE TITLE
New cursor

### ISSUE THREAD

**Issue by geekrelief** [Seneral/TC2-Node-Painter-Beta#25]
We discussed having a halo above the brush preview indicate intensity as well as a line to show Lerp/Replace's Target.

What do you think of having a different cursor?  Instead of a circle it could be simplified so that it won't be confused with the new halo.

For example, here's a simple dot and arrow.  If you could rotate the cursor it could also indicate rotation.  This extra indicator would be a helpful when switching between brushes that are symmetrical and asymmetrical as it's a reminder that rotation has been changed.

![capture](https://cloud.githubusercontent.com/assets/27040/23733716/4c3375f8-042f-11e7-81ff-71db28662b4b.PNG)

**Comment by geekrelief** [comment]
Without a change in brush preview intensity at low levels, it's difficult for the user to visual changes to the terrain.  Having the arrow indicator would also be useful if the preview intensity is too low as at least the rotation of the brush can be visualized.

**Comment by Seneral** [comment]
Yes, plan to atleast increase vizualisation intensity for now - the others, maybe in a later update:)
